### DISCUSSION TITLE
Calendar is not working properly

### DISCUSSION THREAD

**Discussion by Its-Just-Nans** [community/community#74391] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

What we know:
- Github is using Sunday as first day of the week
- during https://en.wikipedia.org/wiki/Leap_year_starting_on_Saturday (aka 1972, 2000 and 2028...) the calendar has 53 weeks !

What is happening:
**So if I render the calendar of the 2000 year, the 1rst day of the year is not displayed 😮**

![image](https://github.com/community/community/assets/56606507/45bde6f8-6c95-497c-bca0-4cb216d550a5)

![image](https://github.com/community/community/assets/56606507/32ad74af-66ca-43dc-96c9-5e6c7b0682bd)

Example : https://github.com/torvalds?tab=overview&from=2000-01-01&to=2000-12-31

> If github were using Monday as first day, the bug will happen during https://en.wikipedia.org/wiki/Leap_year_starting_on_Sunday (aka 2012, 2040...)

**Comment by ThomasHaynes1337** [comment]
Ah, calendars and their quirks! Your post about GitHub using Sunday as the first day of the week brought back memories of my own calendar-related challenges. I remember a time when I was working on a project that required precise date tracking, and a similar issue cropped up.
Back then, I stumbled upon https://calendargeek.com/, a website that turned out to be a goldmine of calendar tricks and tips. It's funny how sometimes these quirks in systems lead us to discover valuable resources. CalendarGeek had a plethora of tutorials on iCal and various calendar-related hacks.

**Reply by Its-Just-Nans** [reply]
The actual bug is not the fact Github calendar is using this or that day to start the week ;)

The bug is that github is not managing **Leap year** correctly
